packages/syft/src/syft/core/node/domain_client.py

```diff
@@ -759,7 +759,9 @@ def load_dataset(
             "\n\nRun `<your client variable>.datasets` to see your new dataset loaded into your machine!"
         )
 
-    def create_user(self, name: str, email: str, password: str, budget: int) -> dict:
+    def create_user(self, name: str, email: str, password: str, budget: float) -> dict:
```

Comment:
I think we should also include a budget  sanity  check in the service side code also  in the service part of 
```py
 def create_user_msg
```
 of user_manager_service.py
https://github.com/OpenMined/PySyft/blob/8f550d8fe6a249d0e956335b033c1870b9870b1f/packages/syft/src/syft/core/node/common/node_service/user_manager/user_manager_service.py#L36
As the  code would also allow data scientist to create custom message with their custom values.

Reply:
I will look a little bit at this! I will come back with an update